homeassistant/components/tuya/sensor.py

```diff
@@ -739,6 +739,74 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             icon="mdi:progress-clock",
         ),
     ),
+    # Air Purifier
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48r41mn81
+    "kj": (
+        TuyaSensorEntityDescription(
+            key=DPCode.FILTER,
+            name="Filter Utilization",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            icon="mdi:ticket-percent-outline",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.FILTER_DAYS,
+            name="Remaining days of the filter",
```

Comment:
Hmm, this is a countdown and should be using a date object of when the countdown ends.

Reply:
About that PR of creating the conversion to timestamp, do you think it will be viable? Well, that would use that too, right?

Reply:
Except that this is not a timestamp, but a date.

Reply:
It's been taken out, I'll insert when the PR implementing this is handled.